package.json

```diff
@@ -14,64 +14,58 @@
   "dependencies": {
     "@aws-sdk/client-kinesis": "3.53.0",
     "@aws-sdk/client-sqs": "3.53.0",
-    "@graphql-tools/mock": "^8.5.0",
-    "@graphql-tools/schema": "^8.3.1",
+    "@graphql-tools/mock": "8.6.3",
```

Comment:
note for future: we can remove

Reply:
I was wondering if we could, but couldnt tell!